### PR TITLE
System as value

### PR DESCRIPTION
**Pull request by talekhinezh**
- Implement epoch as part of system substate
- Remove get/set epoch from substate store

### CODE REVIEW

File: scrypto/src/constants.rs [hunk 1]
```diff
@@ -25,3 +25,8 @@ pub const RADIX_TOKEN: ResourceAddress = ResourceAddress([
 pub const ECDSA_TOKEN: ResourceAddress = ResourceAddress([
     0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 5,
 ]);
+
+/// The system token which only validators have access to
```
Comment (dhedey) [inline scrypto/src/constants.rs:29]: It's not just validators - it's more the system executor, or I think we called them "system transactions", I believe?

File: radix-engine/src/engine/track.rs [hunk 2]
```diff
@@ -251,6 +256,22 @@ impl SubstateValue {
...
+        } else {
+            panic!("Not a system value");
+        }
+    }
+
+    pub fn system_mut(&mut self) -> &mut System {
+        if let SubstateValue::System(system) = self {
+            system
+        } else {
+            panic!("Not a vault");
```
Comment (dhedey) [inline radix-engine/src/engine/track.rs:271]: Not a system value*?

Comment (talekhinezh) [reply radix-engine/src/engine/track.rs:271]: good catch

**Review by iamyulong — APPROVED** [review]
LGTM

**Pull request merged**